shared/editor/rules/toggleBlocks.ts

```diff
@@ -0,0 +1,9 @@
+import MarkdownIt from "markdown-it";
+import customFence from "markdown-it-container";
+
+export default function toggleBlocks(md: MarkdownIt): void {
+  return customFence(md, "toggle_block", {
```

Comment:
```suggestion
  return customFence(md, "toggle", {
```

Lets just call it "toggle" so it ends up `container_toggle`